.github/workflows/security-container-vulerability-scanning.yaml

```
@@ -0,0 +1,91 @@
+          format: 'sarif'
+          output: '${{ matrix.component }}-results.sarif'
+        # env:
+        #   TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db:2
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v4
+        with:
+          sarif_file: '${{ matrix.component }}-results.sarif'
+          category: ${{ matrix.component }}
+
+  analyze:
```

actionlint (1.7.11)

[error] job "analyze" needs job "build_multi-agent" which does not exist in this workflow

(job-needs)